### ISSUE TITLE
Strange symbol on sway cannot be changed !

### ISSUE THREAD

**Issue by catafest** [swaywm/sway#7712]
### Please read the following before submitting:
- We are not accepting any new window management features unless they get implemented by i3.
I don't know very much about your implementation by i3, but if this helps you then need to be fixed.

### Please fill out the following:
- **Description:**
Please describe in plain English what the enhancement is and what the use case is.
I can't change this symbol from sway with the config file, I don't work on the development sway area but looks strange.
![image](https://github.com/swaywm/sway/assets/2210118/25863342-7cab-4ccc-a65a-3c9d2e04de45)

**Comment by mhmdanas** [comment]
This doesn't seem related to Sway at all? It's probably something on your side, or a Waybar issue.

**Comment by vyivel** [comment]
This is an issue with your Waybar configuration, most probably caused by not having the required icon font installed.
